TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -313,15 +301,15 @@ import PromiseKit
         // Add missing linkedBrokers
         let localUserIds = self.linkedBrokers.flatMap { $0.linkedLogin.userId }
         let remoteLinkedBrokersToAdd = remoteLinkedBrokers.filter { !localUserIds.contains($0.userId) }
-        
+
         remoteLinkedBrokersToAdd.forEach { remoteBrokerData in
-            injectBroker(
+            self.saveLinkedBrokerToKeychain(
```

Comment:
These were confusing. I thought initially these were async calls and was about to promisify everything but appears that it's actually sync.